README.md

```diff
@@ -21,6 +21,11 @@ scripts:
   - name: timeout
     script: sleep 5
     timeout: 1
+
+  - name: target
+    script: example.sh
+    timeout: 4
+    target: service.example.com
```

Comment:
While the configuration may make it possible to specify the `target` in a configuration file, I suggest leaving this part of the example out as it obscures the intended usage via environment variable.

Could the `script:` above correctly be shown as:
```
  - name: target
￼    script: example.sh ${TARGET}
    timeout: 4
```

Reply:
Or is the expectation that `example.sh` inspect its environment before running?